Cargo.lock

```diff
@@ -2,7 +2,10 @@
 name = "rstreediff"
 version = "0.1.0"
 dependencies = [
+ "dot 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
The consensus seems to be that binaries should commit their lock files, but libraries should not: http://doc.crates.io/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries